### PR TITLE
rstcheck done

### PR DESCRIPTION
**Pull request by sourabhtk37**
Files changed:

1. README.rst
2. RELEASE_NOTE.rst

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -128,15 +128,6 @@ For more information about how to do basic analysis, check out the
 
 -----
 
-=======
```
Comment (jayvdb) [inline README.rst]: Why was this removed?

File: RELEASE_NOTES.rst [hunk 2]
```diff
@@ -175,8 +175,7 @@ are added for existing bears.
 
 New bears:
 -  ``AutoPrefixBear`` (Add vendor prefixes automatically in CSS)
--  ``ClangComplexityBear`` (Calculates cyclomatic complexity of each function
```
Comment (jayvdb) [inline RELEASE_NOTES.rst]: why was this changed?

File: RELEASE_NOTES.rst [hunk 3]
```diff
@@ -217,13 +216,14 @@ Bugfixes:
...
    (https://github.com/coala-analyzer/coala-bears/issues/251)
 
+
 coala-bears 0.1.0 beta
 =======================
 
 coala-bears is a Python package containing all the bears that are used by coala.
 It has been split from `coala <https://github.com/coala-analyzer/coala>`_.
 With the initial release, it features 56 bears covering 32 languages.
-You can see all of them `here <https://gist.github.com/Adrianzatreanu/cf2d0c8b2ecd542a4860>`_
```
Comment (jayvdb) [inline RELEASE_NOTES.rst]: Does rstcheck recommend this change?
Why?

File: RELEASE_NOTES.rst (hunk 2 above)
Comment (sourabhtk37) [inline RELEASE_NOTES.rst]: rstcheck revealed difference in indents so I made sure they were uniform.Nothing else.

File: RELEASE_NOTES.rst (hunk 3 above)
Comment (sourabhtk37) [inline RELEASE_NOTES.rst]: Actually this change, i was a bit skeptical about.Not necessary as such but having a little OCD of seeing a rstcheck reveal that there were two instances of "here"(hypertext-link) in files,so I changes it.

File: README.rst (hunk 1 above)
Comment (jayvdb) [inline README.rst]: Excellent.  This should be described in the commit message.

File: RELEASE_NOTES.rst (hunk 2 above)
Comment (jayvdb) [inline RELEASE_NOTES.rst]: > Nothing else.

Not true ;-)

You moved the indented text to the previous line.  That makes it a very long line, which isnt acceptable.

I think the required fix is to remove one space from the indent. on the original line 179, so it is aligned with the other text.

File: RELEASE_NOTES.rst (hunk 3 above)
Comment (jayvdb) [inline RELEASE_NOTES.rst]: I believe that rstcheck is being very pedantic if it is warning about this.

However "here" is overused and disliked as a link text.  But "changes here" is not a huge improvement.

"on this gist" might be better, as it gives the reader a little more information about what the link is.

Comment (sourabhtk37) [inline RELEASE_NOTES.rst]: Cool.

**Comment by Adrianzatreanu** [comment]
could you please address gitmate's comments, especially the ones regarding your commit message? thanks.

**Comment by Adrianzatreanu** [comment]
also, @jayvdb this is a PR, not an issue, why did you add difficulty label?

**Comment by jayvdb** [comment]
@Adrianzatreanu, this was originally incorrectly labeled as a PR for #902 ; as that is not correct, and there is no issue for this PR, I added a few issue-ish labels.  Remove them if they break anything.

File: RELEASE_NOTES.rst (hunk 2 above)
Comment (jayvdb) [inline RELEASE_NOTES.rst]: Please look at your patch *very carefully*.
You have moved "` for C,`" to the previous line.
Please undo that change.

**Comment by jayvdb** [comment]
@sourabhtk37, please squash your [three commits](https://github.com/coala/coala-bears/pull/904/commits) into a single commit, with one commit message. See http://coala.readthedocs.io/en/latest/Developers/Git_Basics.html

File: README.rst (hunk 1 above)
Comment (jayvdb) [inline README.rst]: I have created a new issue about this problem:
https://github.com/coala/coala-bears/issues/918

Please include `Fixes https://github.com/coala/coala-bears/issues/918` in your commit message

File: RELEASE_NOTES.rst (hunk 2 above)
Comment (jayvdb) [inline RELEASE_NOTES.rst]: https://github.com/coala/coala-bears/issues/919 has been created about this problem.  Please mention it in your commit message.
